docs/source/tusupload.rst

```diff
@@ -1,88 +1,99 @@
 TUS resumable upload
 ====================
 
-Plone REST API supports the TUS Open Protocol for Resumable File Uploads.
-See http://tus.io/ for more information on TUS.
+plone.restapi supports the `TUS Open Protocol <http://tus.io>`_ for resumable file uploads.
+The two TUS endpoints to upload '@tus-upload' a file and to replace an existing file '@tus-replace' are available on all 'folderish resources' in Plone (content types that implement the IFolderish interface, e.g. a 'Folder').
```

Comment:
@buchi fixed in the docs. Though, I do not fully understand why one would want to use tus-replace without tus-upload on a non-folderish object. My understanding of TUS is limited though. :)